torchmetrics/audio/pit.py

```diff
@@ -0,0 +1,114 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Dict, Optional
+
+from torch import Tensor, tensor
+
+from torchmetrics.functional.audio.pit import pit
+from torchmetrics.metric import Metric
+
+
+class PIT(Metric):
```

Comment:
is it only for audio or rather a wrapper for other metrics, see
https://github.com/PyTorchLightning/metrics/blob/591bb2d7f92985efb321b54f96b2628c79e56fe5/torchmetrics/wrappers/bootstrapping.py#L49

Reply:
It is only for audio. The PIT implements the famous Permutation Invariant Training method in speech separation field in order to calculate audio metrics in a permutation invariant way.

Reply:
@quancs could you add what you just wrote to the docstring? seems great to establish what it is used for

Reply:
done. see https://github.com/PyTorchLightning/metrics/pull/384/commits/01d484edabd99d35c988c6615845edb00d5bd40c